en/components/grids_templates/excel-style-filtering.md

```diff
@@ -587,15 +593,15 @@ In this example we only changed some of the parameters for the listed components
 The last step is to **include** the component mixins, each with its respective theme. We will also set the color property for the input's placeholder.
```

Comment:
The description is incorrect, css-vars is not a component mixin! It's a mixin that adds theme colors as CSS variables to a given scope.